app.rb

```diff
@@ -55,30 +56,34 @@ def authorized?
         url: config.idp_url,
         headers: { 'Authorization' => "Bearer #{config.attempts_api_csp_id} #{irs_attempt_api_auth_token}" }
       )
-      body = "timestamp=#{Time.now.iso8601}"
+      Time.zone = "UTC"
```

Comment:
this should not happen inside a request, and I think safer code would be to reference specific timezones

```
utc = ActiveSupport::TimeZone['UTC']
now = utc.now
```

etc

Reply:
"this" = the side effect of setting a global like this